### DISCUSSION TITLE
QChat with images and attachments

### DISCUSSION THREAD

**Discussion by juuunior-net** [quasarframework/quasar#10757] [General - Components / Directives / etc]
Hey folks, is it possible to have Images and Attched docs in the QChatMessage ?

**Comment by ontwikkelfabriek** [comment]
@juuunior-net 

Not out of the box, but you can use `slots` to `customize` the QChatMessage:

Look at the `slots` section of the QChatMessageAPI:
https://quasar.dev/vue-components/chat#qchatmessage-api 

Here's an example of a customized QChatMessage
https://quasar.dev/vue-components/chat#slots
